File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/client/paths/pet/put/security/PetPutSecurityRequirementObject0.java
package org.openapijsonschematools.client.paths.pet.put.security;

import org.openapijsonschematools.client.securityrequirementobjects.SecurityRequirementObject;
import org.openapijsonschematools.client.securityschemes.SecurityScheme;

import java.util.List;
import java.util.Map;

public class PetPutSecurityRequirementObject0 extends SecurityRequirementObject {
    // todo make n specific constructors
    public PetPutSecurityRequirementObject0(Map<SecurityScheme, List<String>> securitySchemeToScopes) {
        super(securitySchemeToScopes);
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/paths/pet/put/security/PetPutSecurityRequirementObject0.kt
package org.openapijsonschematools.client.paths.pet.put.security;

import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.securityrequirementobjects.SecurityRequirementObject;
import org.openapijsonschematools.client.securityschemes.SecurityScheme;
import org.openapijsonschematools.client.components.securityschemes.HttpSignatureTest;

import java.util.List;
import java.util.Map;
import java.util.AbstractMap;

public class PetPutSecurityRequirementObject0 extends SecurityRequirementObject {

    public PetPutSecurityRequirementObject0() {
        super(
            Map.ofEntries(
                new AbstractMap.SimpleEntry<Class<? extends SecurityScheme>, List<String>>(
                    HttpSignatureTest.class,
                    List.of()
                )
            )
        );
    }
}
